doc/modules/compose.rst

```diff
@@ -570,3 +571,31 @@ As an alternative, the HTML can be written to a file using
 
  * :ref:`sphx_glr_auto_examples_compose_plot_column_transformer.py`
  * :ref:`sphx_glr_auto_examples_compose_plot_column_transformer_mixed_types.py`
+
+Visualizing a Chain of Estimators
+==================================
+
```

Comment:
This section looks to be the same as the section above it "Visualizing Composite Estimators". Was this section meant to be included?